readme.html

```diff
@@ -72,53 +78,236 @@ <h2>Installation</h2>
     automatically deployed. To upgrade to a new version, copy the new httpfileupload.jar file over the existing file.
 </p>
 
-<h2>Configuration</h2>
+<h2>Using the Plugin</h2>
+
+<p>
+    After installation, the functionality provided by the plugin is automatically available to clients. While exchanging
+    files, the plugin by default stores the files that are being transferred in a temporary directory that is removed
+    when Openfire is shut down. The content of this directory is purged when its total size is larger than the remaining
+    disc space.
+</p>
+
+<h2 id="configuration">Configuration</h2>
+
+<p>
+    Most configuration options can be controlled by two different properties. Apart from using regular Openfire properties,
+    controlled in the Openfire admin console, XML properties can be used. XML properties are properties stored in
+    the <samp>openfire.xml</samp> file.
+</p>
+<p>
+    Regular properties are stored in the database. This will ensure that a property value is equal on all
+    servers in a cluster of Openfire servers. When it is desirable to have a property values that are <em>different</em>
+    on each server, an XML property can be used (as the content of the <samp>openfire.xml</samp> file can be different
+    on each server).
+</p>
+<p>
+    When both a regular and an XML property is defined, then the XML property value will take precedence.
+</p>
+
+<h3>Storage of data on filesystem</h3>
+
+<table>
+    <tr><th>XML property</th> <td><samp>plugin.httpfileupload.serverspecific.fileRepo</samp></td></tr>
+    <tr><th>Property</th> <td><samp>plugin.httpfileupload.fileRepo</samp></td></tr>
```

Comment:
Should this be System Property, to be super-specifc?
```suggestion
    <tr><th>System Property</th> <td><samp>plugin.httpfileupload.fileRepo</samp></td></tr>
```